README.md

```diff
@@ -2,3 +2,20 @@
 
 A buildpack for including the ICU library into a Heroku dyno. This is required
 as a dependency for several tools.
+
+## How it works
+
+This buildpack makes ICU 4.8 available in the `/app/icu` directory for both
+compilation and runtime uses. The primary purpose of this is to allow Charlock
```

Comment:
Compilation is within the slug compiler (so while buildpacks are running). During this process, anything you need in your build needs to go into the directory passed as `$1` to the script. This directory is frequently something arbitrary like `/tmp/<random-string>/`.

At runtime (so when the compiled slug is started), anything you placed in the build directory will be available under `/app` of the dyno's file system.

As Charlock Holmes uses dynamic linking, the compiled extension it provides compiles in the directory of the ICU library... as this will be `/app/icu` at runtime, we need to make sure that the library is available there at compile time too, hence this step.